index.html

```diff
@@ -12,5 +12,13 @@
       crossorigin="anonymous"
     />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="container-fluid">
+          <a class="navbar-brand" container-fluid href="/">Oly Su</a>
```

Comment:
```suggestion
          <a class="navbar-brand" href="/">Oly Su</a>
```

Please remove `container-fluid` attribute.